### PR TITLE
Feature/kitosudv 2468

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Web.config [hunk 1]
```diff
@@ -22,7 +22,7 @@
   </configSections>
   <connectionStrings>
     <add name="KitosContext"
-         connectionString="Server=.\SQLEXPRESS;Integrated Security=true;Initial Catalog=Kitos;MultipleActiveResultSets=True"
+         connectionString="Server=.\SQLEXPRESS;Integrated Security=true;Initial Catalog=Kitosproduction;MultipleActiveResultSets=True"
```
Comment (mrjsawdk) [inline Presentation.Web/Web.config:25]: remember to revert

File: Core.DomainModel/Shared/LifeCycleEventType.cs [hunk 2]
```diff
@@ -4,6 +4,6 @@ public enum LifeCycleEventType
     {
         Created = 0,
         Updated = 1,
-        Deleted = 2
+        Deleting = 2
```
Comment (mrjsawdk) [inline Core.DomainModel/Shared/LifeCycleEventType.cs:7]: check if this results in issues on the api

File: Core.DomainServices/Context/DefaultOrganizationResolver.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+        public DefaultOrganizationResolver(IOrganizationRepository organizationRepository, ILogger logger)
+        {
+            _organizationRepository = organizationRepository;
+            _logger = logger;
+        }
+
+        public Organization Resolve()
+        {
+            var organizations = _organizationRepository.GetAll().Where(x => x.IsDefaultOrganization == true).ToList();
```
Comment (mrjsawdk) [inline Core.DomainServices/Context/DefaultOrganizationResolver.cs:22]: check if it is the isqueryable..

File: Tests.Integration.Presentation.Web/Organizations/OrganizationSequentialTests.cs [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.External;
+using Tests.Integration.Presentation.Web.Tools.XUnit;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Organizations
+{
+    [Collection(nameof(SequentialTestGroup))]
+
```
Comment (mrjsawdk) [inline Tests.Integration.Presentation.Web/Organizations/OrganizationSequentialTests.cs:17]: unneeded newline

File: Core.DomainServices/Context/DefaultOrganizationResolver.cs (hunk 3 above)
Comment (mrjsawdk) [reply Core.DomainServices/Context/DefaultOrganizationResolver.cs:22]: it is

File: Core.DomainModel/Shared/LifeCycleEventType.cs (hunk 2 above)
Comment (mrjsawdk) [reply Core.DomainModel/Shared/LifeCycleEventType.cs:7]: no issues. It's correct in the context where it is used. The tracking db has a different enum and the api as well

File: Presentation.Web/Models/API/V1/Organizations/InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿namespace Presentation.Web.Models.API.V1.Organizations
+{
+    public class InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO
```
Comment (a-naskret) [inline Presentation.Web/Models/API/V1/Organizations/InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO.cs:3]: I'm wondering what is the purpose of this class, and what is it used for?

Comment (mrjsawdk) [reply Presentation.Web/Models/API/V1/Organizations/InterfaceExposedOnSystemsOutsideTheOrganizationConflictDTO.cs:3]: @a-naskret It's used on the main conflicts DTO:
image

**Review by a-naskret — APPROVED** [review]

**Pull request merged**